pkg/storage/couchdb/couchdbstore.go

```diff
@@ -234,8 +234,7 @@ func (c *CouchDBStore) Put(k string, v []byte) error {
 	}
 
 	revID, err := c.getRevID(k)
-	if err != nil && !errors.Is(err, storage.ErrValueNotFound) &&
-		!strings.Contains(err.Error(), docDeletedErrMsgFromKivik) {
+	if err != nil && !errors.Is(err, storage.ErrValueNotFound) {
```

Comment:
nit: `errors.Is(err, storage.ErrValueNotFound)` takes care of matching `err != nil`